### ISSUE TITLE
Environments for this Repo. (Anyone succeeded to reproduce the results.)

### ISSUE THREAD

**Issue by peter-wang321** [JiaDingCN/GeminiFusion#8]
Great work! Thanks for releasing the code. However, I cannot run the code successfully based on the instructions. Have you tested it by rebuilding a new conda environment? If you succeed in reproducing the results based on this repo., could you provide me with environments by running the following commands?

$ python3 -c "import torch; print(torch.\_\_version\_\_)"
$ python3 -V
$ pip list

Thanks a lot!

**Comment by peter-wang321** [comment]
I am very grateful for the author's response to my questions, which gave me important insights on reproducing the results of this repo. 

And now I did it. The following are my settings about the environment, which is more important to keep it exactly consistent.

[GeminiFusion]$ python3 -c "import torch; print(torch.__version__)"
1.8.0+cu111
[GeminiFusion]$ python3 -V
Python 3.6.13 :: Anaconda, Inc.
[GeminiFusion]$ pip list
Package             Version
------------------- -----------
addict              2.4.0
certifi             2021.5.30
charset-normalizer  2.0.12
cycler              0.11.0
dataclasses         0.8
filelock            3.4.1
huggingface-hub     0.4.0
idna                3.7
importlib-metadata  4.8.3
importlib-resources 5.4.0
kiwisolver          1.3.1
matplotlib          3.3.4
mmcv-full           1.6.0
mmdet               2.25.0
numpy               1.19.5
opencv-python       4.5.1.48
packaging           21.3
Pillow              8.4.0
pip                 21.2.2
pycocotools         2.0.7
pyparsing           3.1.2
python-dateutil     2.9.0.post0
PyYAML              6.0.1
requests            2.27.1
setuptools          58.0.4
six                 1.16.0
terminaltables      3.1.10
timm                0.6.12
torch               1.8.0+cu111
torchaudio          0.8.0
torchvision         0.9.0+cu111
tqdm                4.64.1
typing_extensions   4.1.1
urllib3             1.26.19
wheel               0.37.1
yapf                0.32.0
zipp                3.6.0

Besides the environments, I also made two changes: First, deleting the "batch_first" since it is not supported in pytorch 1.8.0, and it is "batch_first=False" in the code. Second, replace (np.bool) with (bool) since it is not supported by Numpy 1.19.5. 

That is what I did to reproduce it. I am not sure whether the changes will incur other potential issues, but now I have reproduced the results as reported in the Repo.

Many thanks to the authors.
